src/labels/CustomLabels.labels

```diff
@@ -1951,6 +1951,23 @@
         &lt;br/&gt;&lt;br/&gt;&lt;b&gt;Because you have no Recurring Donation records, you don&apos;t need to validate or migrate any data.&lt;/b&gt;
         &lt;br/&gt;&lt;br/&gt;For more information, read the &lt;a href=&quot;https://powerofus.force.com/NPSP_Recurring_Donations&quot; target=&quot;_blank&quot;&gt;Enhanced Recurring Donations documentation&lt;/a&gt;.</value>
     </labels>
+
+    <labels>
+        <fullName>RD2_EndDateMustBeLaterThanCloseWonOpp</fullName>
+        <categories>Recurring Donations Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Recurring Donation End Date must be valid.</shortDescription>
+        <value>End Date must be later than the last Closed Won Opportunity&apos;s Close Date.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_EndDateMustBeValidForFixedLength</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Fixed Length Recurring Donation End Date must be valid</shortDescription>
+        <value>End Date can&apos;t be in the future when Recurring Type is Fixed.</value>
+    </labels>
```

Comment:
![Screen Shot 2020-06-01 at 12 13 55 PM](https://user-images.githubusercontent.com/57271120/83446364-64ba2780-a403-11ea-9004-8c11560e38da.png)
![Screen Shot 2020-06-01 at 12 09 55 PM](https://user-images.githubusercontent.com/57271120/83446365-6552be00-a403-11ea-9271-2077eb2df9fb.png)

@sam-knox Please review the 2 new error messages with EndDate.

Reply:
@andrewyu-salesforce - Yep, those both look good!